docs/troubleshooting.md

```diff
@@ -149,3 +149,38 @@ releases (including automated upgrades) it has applied, and that only
 matters if it has been asked to report those with the `--connect`
 flag. Future versions of Flux may be more sparing in use of the sync
 tag.
+
+### Flux fails with an error log similar to _couldn't get resource list for example.com/version: the server is currently unable to handle the request_
+
+This means your Kubernetes cluster fails to respond to list queries
+for resources in _example.com/version_.
+
+If the error is transient, Flux will work once the error recedes.
+
+However, the error won't normally go away by itself since it's normally
+caused by a misconfiguration of your cluster.
+
+For instance, you can run into this problem when a
+[Kubernetes Webhook server](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
+is removed without removing its Webhook definition.
```

Comment:
Please add:
Or when a custom resource definition is not available due to `FailedDiscoveryCheck` error.